github-fetcher/json-orm.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
https://blog.golang.org/package-names

Try to use neither. Usually this file is called "types.go" or something like that.